packages/ui-demo/src/TextField.stories.tsx

```diff
@@ -1,9 +1,12 @@
+import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
 import {Text, TextField} from "ferns-ui";
-import moment from "moment-timezone";
 import React, {ReactElement, useState} from "react";
 
 import {StorybookContainer} from "./StorybookContainer";
 
+dayjs.extend(utc);
```

Comment:
Do we still need this, or can we use the extended one below?

Reply:
This is just because it's within `ui-demo` and our wrapper is scoped to `ui`.